docs/UserGuide.md

```diff
@@ -62,111 +74,128 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
+  <pre>
+    <code>
+        1. add t/e d/TehBeng a/1.50 da/22-10-2022 c/Food 
+        2. add t/e d/TehBeng a/1.50 da/22-10-2022 c/Food
+    </code>
+  </pre>
 
-Adds an income entry.
+</div>
 
-1. Add income with default current date
-   * Example: `add -i 15.60`
-   * An income of $15.60 will be added with the current date
-   * Expected: `An income of 15.60 on 27 Sep 2022 has been recorded`
-2. Add income with specific date
-   * Example: `add -i 15.60 / 12-09-2022`
-   * An income of $15.60 will be added with 12 September 2022 as the date
-   * Expected: `An income of 15.60 on 12 Sep 2022 has been recorded`
+### Adding entries : `add t/ENTRY_TYPE d/DESCRIPTION a/AMOUNT da/DATE c/CATEGORY`
+Adds an entry to the specified list. <br>
+1. Add expense <br>
+   - Example: `add t/e d/Lunch a/15.60 da/10-10-2022 c/Food`<br>
+   - An expense of 15.60 on 10/10/2022 will be added to the Expenditure list, under the Food category.
+
+1. Add income <br>
+   - Example: `add t/i d/Tuition a/40.00 da/10-10-2022 c/Salary`<br>
+   - An income of 40.00 on 10/10/2022 will be added to the Income list, under the Salary category.
+
+### Deleting entries : `del INDEX_OF_ENTRY t/ENTRY_TYPE`
+Deletes an entry.
+1. Example: `del 2 t/e` deletes the 2nd item on the expenditure list 
+   Expenditure list:
+   1. Movie 15.60 12 Sep 2022 c/Entertainment
+   2. Drink 1.20 12 Sep 2022 c/Food 
+   Expected: `Deleted Entry: Drink; Date: 12-09-2022; Amount: 1.20; Tag: Food`
+   
+1. Example: `del 2 t/i` deletes the 2nd item on the income list
+   Income list:
+   1. Tuition 40.00 12 Sep 2022 c/Salary
+   2. Allowance 100.00 12 Sep 2022 c/Allowance
+   Expected: `Deleted Entry: Allowance; Date: 12-09-2022; Amount: 100.00 Tag: Allowance`
+
+
+### Editing entries : `edit INDEX_OF_ENTRY t/ENTRY_TYPE [d/EDITED_DESCRIPTION a/EDITED_AMOUNT da/EDITED_DATE c/EDITED_CATEGORY]`
+Edits an entry, at least **1** of the optional fields must be present.
+
+1. Edits expenditure with specified fields
+   * Example: `edit 1 t/e d/ChickenRice`
+   * The expenditure at index 1 will have its description edited
+   * Expected: `Edited Entry: ChickenRice; Date: 21-10-2022; Amount: 4.20; Tag: Food`
+1. Edits income with specified fields
+   * Example: `edit 2 t/i a/150.00 da/22-10-2022`
+   * The income at index 2 will have its amount and date edited
+   * Expected: `Edited Entry: CafeSalary; Date: 22-10-2022; Amount: 150.00; Tag: Salary`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 The default date is the current date on your computer!
 </div>
 
-### Deleting income: `del -i NUMBER INDEX_OF_INCOME`
-
-Deletes an income entry
-
-Example: `del -i 2` deletes the 2nd item on the list
-
-Spending list:
-1. Earned 5.00 12 Sep 2022
-2. Earned 30.00 12 Sep 2022
-
-Expected: `An income of 30.00 on 12 Sep 2022 has been deleted.`
-
+### Summary of entries : `summary [mo/MONTH]`
 
-### Viewing total summary of spendings : `view -s [MONTH]`
+1. View a summary of all entires
```

Comment:
Should we change word view to avoid confusion w view command? maybe "Summarise all entries" and "Summarise all entries in specified month"

Reply:
have changed it to **Summarise all entires...** instead